### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by haesleinhuepf**
Hi Johannes @jo-mueller ,

here comes a readme update featuring
* Basic tutorial first (files yet missing)
* Simplified some text explaining what receipes are
* Added badges on top of page
* Added DOI on zenodo
* Simplified installation instructions

Let me know what you think or feel free to merge!

Thanks!

Best,
Robert

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,66 +1,92 @@
+[![License](https://img.shields.io/pypi/l/napari-stress.svg?color=green)](https://github.com/biapol/napari-stress/raw/master/LICENSE)
+[![PyPI](https://img.shields.io/pypi/v/napari-stress.svg?color=green)](https://pypi.org/project/napari-stress)
+[![Python Version](https://img.shields.io/pypi/pyversions/napari-stress.svg?color=green)](https://python.org)
+[![Anaconda-Server Badge](https://anaconda.org/conda-forge/napari-stress/badges/version.svg)](https://anaconda.org/conda-forge/napari-stress)
```
Comment (jo-mueller) [inline README.md:4]: This one doesn't display for me - https://anaconda.org/conda-forge/napari-stress requires a login?

File: README.md [hunk 2]
```diff
@@ -1,66 +1,92 @@
...
-
-![](./docs/imgs/function_gifs/spherical_harmonics.gif)
+![](https://github.com/BiAPoL/napari-stress/raw/main/docs/imgs/function_gifs/spherical_harmonics.gif)
 
 ## Usage
 
 Functionality in this repository is divided in two groups: **Recipes** and **plugins**.
 
-### Plugins
+### Receipse
```
Comment (jo-mueller) [inline README.md:24]:
```suggestion
### Recipes

```

File: README.md [hunk 3]
```diff
@@ -1,66 +1,92 @@
...
 
-All functions in this repository are documented separately for [interactive usage from the napari viewer](./docs/tutorials/point_and_click) as well as [Jupyter notebooks](./docs/tutorials/demo). 
+Receipes are workflows for processing images, points and surface data step-by-step.
 
-|Function| Links |
+| Receipe | Description |
```
Comment (jo-mueller) [inline README.md:28]:
```suggestion
| Recipe| Description |

```

**Review by jo-mueller — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (haesleinhuepf) [reply README.md]: Fixed. I removed the button. We should bring it back once the plugin is on conda

**Review by jo-mueller — APPROVED** [review]

**Pull request merged**